snuba/datasets/querylog.py

```diff
@@ -0,0 +1,94 @@
+from datetime import timedelta
+from snuba import settings
```

Comment:
We generally leave a blank line between system imports and snuba imports, so here and not at line 17